pkgs/os-specific/linux/minimal-bootstrap/python/default.nix

```diff
@@ -0,0 +1,97 @@
+{
+  lib,
+  buildPlatform,
+  hostPlatform,
+  fetchurl,
+  bash,
+  gcc,
+  musl,
+  binutils,
+  gnumake,
+  gnupatch,
+  gnused,
+  gnugrep,
+  gawk,
+  diffutils,
+  findutils,
+  gnutar,
+  xz,
+  zlib,
+}:
+let
+  pname = "python";
+  version = "3.13.7";
+
+  src = fetchurl {
+    url = "https://www.python.org/ftp/python/${version}/Python-${version}.tar.xz";
+    hash = "sha256-VGL5CZ39MOI43vg8cdkYl9jKpf9uvHpQ8U1IAs2qp5o=";
+  };
+
+  patches = [
+    # Disable the use of ldconfig in ctypes.util.find_library (since
+    # ldconfig doesn't work on NixOS), and don't use
+    # ctypes.util.find_library during the loading of the uuid module
+    # (since it will do a futile invocation of gcc (!) to find
+    # libuuid, slowing down program startup a lot).
+    ./no-ldconfig.patch
+  ];
+in
+bash.runCommand "${pname}-${version}"
+  {
+    inherit pname version;
+
+    nativeBuildInputs = [
+      gcc
+      musl
+      binutils
+      gnumake
+      gnupatch
+      gnused
+      gnugrep
+      gawk
+      diffutils
+      findutils
+      gnutar
+      xz
+    ];
+
+    passthru.tests.get-version =
+      result:
+      bash.runCommand "${pname}-get-version-${version}" { } ''
+        ${result}/bin/python3 --version
+        mkdir $out
+      '';
+
+    meta = {
+      description = "A high-level dynamically-typed programming language";
+      homepage = "https://www.python.org";
+      license = lib.licenses.psfl;
+      mainProgram = "python3";
+      platforms = lib.platforms.linux;
+      teams = [ lib.teams.minimal-bootstrap ];
+    };
+  }
+  ''
+    # Unpack
+    tar xf ${src}
+    cd Python-${version}
+
+    # Patch
+    ${lib.concatMapStringsSep "\n" (f: "patch -Np1 -i ${f}") patches}
+
+    # Configure
+    export CC=musl-gcc
+    export C_INCLUDE_PATH="${zlib}/include"
+    export LIBRARY_PATH="${zlib}/lib"
+    export LD_LIBRARY_PATH="$LIBRARY_PATH"
```

Comment:
For reasons I don't completely understand, this build fails on x86_64 in the install phase, because Python fails to load zlib at runtime (via `import zlib`). Luckily I had seen this issue earlier, and it turns out that the following added line seems to fix it:
```bash
export LDFLAGS="-Wl,-rpath,${zlib}/lib -L${zlib}/lib"
```